graphql_test.go

```diff
@@ -79,6 +87,10 @@ type cachedDirectiveVisitor struct {
 	Key string
 }
 
+func (v *cachedDirectiveVisitor) ImplementsDirective(name string) bool {
+	return "cached" == name
```

Comment:
doesn't it read more naturally if the variable is on the left side of the `==` operator?